da/da.md

```diff
@@ -29,9 +29,9 @@ See [da implementation]
 
 [2] [celestia-da][celestia-da]
 
-[3] [grpc-proxy][grpc-proxy]
+[3] [json-rpc-proxy][json-rpc-proxy]
 
 [da implementation]: https://github.com/rollkit/rollkit/blob/main/da/da.go
 [go-da]: https://github.com/rollkit/go-da
 [celestia-da]: https://github.com/rollkit/celestia-da
-[grpc-proxy]: https://github.com/rollkit/go-da/tree/main/proxy
+[json-rpc-proxy]: https://github.com/rollkit/go-da/tree/main/proxy
```

Comment:
I think the link needs to be updated to based on the final updates on the blocking PR. 
for example
```suggestion
[json-rpc-proxy]: https://github.com/rollkit/go-da/tree/main/proxy/json-rpc
```

Reply:
Updated.